### PR TITLE
Sets KEEP_TOGETHER appearance_flag for humans, energy draining text updated to DisplayPower

### LINKED ISSUE #24151: Ninjas using their cloak have their socks visible
image

be2b80ca78f5af791a3a6d36eb5cbfcc1194b185 

RIP ninja invisibility.

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #24151

### CODE REVIEW

**Comment by Qbopper** [comment]
>undershit

**Comment by ShizCalev** [comment]
honk

File: code/modules/ninja/suit/suit.dm [hunk 1]
```diff
@@ -36,6 +36,9 @@ Contents:
 	var/obj/item/clothing/head/helmet/space/space_ninja/n_hood
 	var/obj/item/clothing/shoes/space_ninja/n_shoes
 	var/obj/item/clothing/gloves/space_ninja/n_gloves
+	var/stored_undershirt = null
+	var/stored_underwear = null
+	var/stored_socks = null
```
Comment (Cyberboss) [inline code/modules/ninja/suit/suit.dm]: Remove = nulls please

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/ninja/suit/suit.dm [hunk 2]
```diff
@@ -93,6 +96,24 @@ Contents:
 	s_bombs = rand(5,20)
 	a_boost = rand(1,7)
 
+//Hides the ninja's underclothing to prevent seeing union jack pantyhose when they enter stealth mode.
+/obj/item/clothing/suit/space/space_ninja/proc/update_underclothing(mob/living/carbon/human/H, suit_locked)
+	switch(suit_locked)
+		if(TRUE)
```
Comment (Cyberboss) [inline code/modules/ninja/suit/suit.dm]: This should just be an if

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by AnturK** [comment]
Not sure you noticed when i posted on irc, but alternative here is to set KEEP_TOGETHER on humans. Would need to be tested for side-effects though.

**Comment by ShizCalev** [comment]
@AnturK Gave it a try, the underclothing is still visible when the ninja stealths w/ the KEEP_TOGETHER flag set.

ofih3ew
i5lleqa

**Pull request merged**